core/shared/src/test/scala/cats/parse/ParserTest.scala

```diff
@@ -1093,19 +1119,13 @@ class ParserTest extends munit.ScalaCheckSuite {
   }
 
   property("rep0 is consistent with rep") {
-    forAll(ParserGen.gen, Gen.choose(0, Int.MaxValue), Arbitrary.arbitrary[String]) {
-      (genP, min0, str) =>
-        val min = min0 & Int.MaxValue
-        val repA = genP.fa.rep0(min)
-        val repB = genP.fa
-          .rep(min)
```

Comment:
this was a bug in the test: passing 0 here will throw, and that was possible before.

The solution is to start at 1.